lang/syn/src/codegen/accounts/constraints.rs

```diff
@@ -1060,6 +1063,23 @@ impl<'a> OptionalCheckScope<'a> {
     }
 }
 
+fn generate_get_token_account_space(mint: &Expr) -> proc_macro2::TokenStream {
+    quote! {
+        let mint_info = #mint.to_account_info();
+        if mint_info.owner == anchor_spl::token_2022::Token2022::id() {
```

Comment:
`Token2022` does not exist - but is a good name for a `Program` impl

Reply:
Same with this one, will do!